app/models/like.rb

```diff
@@ -0,0 +1,10 @@
+class Like < ApplicationRecord
+  belongs_to :user, class_name: 'User'
+  belongs_to :post, class_name: 'Post'
+
+  after_create :update_likes_counter
+
+  def update_likes_counter
+    post.increment!(:likes_counter)
+  end
```

Comment:
- [OPTIONAL] Good job by one of the [Rails Active Callbacks](https://guides.rubyonrails.org/active_record_callbacks.html), but after using it you should make the `update_likes_counter` method private. Also, do the same for `update_comments_counter` and `update_posts_counter` methods 🙃 🙃

Reply:
Thank you 😊.